contracts/sol/MerkleTree.sol

```diff
@@ -185,3 +194,50 @@ contract MerkleTree is Ownable, Hasher {
         return nextLeafIndex;
     }
 }
+
+contract EmptyMerkleTreeRoots {
```

Comment:
How expensive is it to calculate in the contract?
I don't think it's critical, it seems fine to me either way, just not sure it's worth the savings.

Reply:
Using a pre-calculated root: 4730825 gas

Using a on-chain calculated root (for a 4-level vote option tree): 5171319 gas

10 levels: 5755717 gas
20 levels: 6729607 gas

Good point, I think it's not worth the savings!